chef/data_bags/crowbar/migrate/glance/012_add_store_admin_keyring.rb

```diff
@@ -0,0 +1,9 @@
+def upgrade ta, td, a, d
+  a['rbd']['store_admin_keyring'] = ta['rbd']['store_admin_keyring']
```

Comment:
If we make ```store_admin_keyring``` optional this is not needed. (The downgrade migration would still be needed though)